modin/data_management/functions/groupby_function.py

```diff
@@ -137,17 +212,53 @@ def reduce(
         return pandas.DataFrame(result)
 
     @classmethod
-    def caller(
+    def groupby_reduce_function(
         cls,
         query_compiler,
         by,
         axis,
         groupby_args,
         map_args,
         map_func,
+        reduce_func,
+        reduce_args,
         numeric_only=True,
-        **kwargs,
+        drop=False,
+        method=None,
     ):
+        """
+        Execute GroupBy aggregation with MapReduce approach.
+
+        Parameters
+        ----------
+        query_compiler : BaseQueryCompiler
+            Frame to group.
+        by : BaseQueryCompiler, column or index label, Grouper or list of such
+            Object that determine groups.
+        axis : {0: Index, 1: Columns}
```

Comment:
same comment about type and description as for other similar parameters in functions above :)